backend/sirius-web-emf/src/main/java/org/eclipse/sirius/web/emf/view/ViewConverter.java

```diff
@@ -122,25 +122,28 @@ public ViewConverter(List<IJavaServiceProvider> javaServiceProviders, IObjectSer
      * {@link DiagramDescription}s are supported. <b>Warning:</b> this code is not re-entrant.
      */
     public List<IRepresentationDescription> convert(View view) {
-        // @formatter:off
-        List<Class<?>> serviceClasses = this.javaServiceProviders.stream()
-                                            .flatMap(provider -> provider.getServiceClasses(view).stream())
-                                            .collect(Collectors.toList());
-        // @formatter:on
-        AQLInterpreter interpreter = new AQLInterpreter(serviceClasses, List.of());
-        try {
+        List<IRepresentationDescription> result = List.of();
+        if (Boolean.getBoolean("org.eclipse.sirius.web.features.studioDefinition")) { //$NON-NLS-1$
```

Comment:
Why don't you use Spring properties system as done everywhere else instead of an environment variable?

Reply:
Done.